src/Files.App/Views/LayoutModes/ColumnViewBase.xaml.cs

```diff
@@ -572,18 +544,23 @@ private void FileList_ItemTapped(object sender, TappedRoutedEventArgs e)
 				{
 					CheckRenameDoubleClick(clickedItem.DataContext);
 				}
-				else if (IsRenamingItem)
+				else if (IsRenamingItem &&
+					FileList.ContainerFromItem(RenamingItem) is ListViewItem listViewItem &&
+					listViewItem.FindDescendant("ListViewTextBoxItemName") is TextBox textBox)
 				{
-					if (FileList.ContainerFromItem(RenamingItem) is ListViewItem listViewItem
-						&& listViewItem.FindDescendant("ListViewTextBoxItemName") is TextBox textBox)
-					{
-						CommitRename(textBox);
-					}
+					CommitRename(textBox);
 				}
-				if (item is not null && item.PrimaryItemAttribute == StorageItemTypes.Folder &&
-					UserSettingsService.FoldersSettingsService.ColumnLayoutOpenFoldersWithOneClick)
+
+				if (isItemFolder && UserSettingsService.FoldersSettingsService.ColumnLayoutOpenFoldersWithOneClick)
+				{
+					ItemInvoked?.Invoke(new ColumnParam { NavPathParam = (item is ShortcutItem sht ? sht.TargetPath : item!.ItemPath), ListView = FileList }, EventArgs.Empty);
```

Comment:
I think we should use single line if statements without curly braces

Reply:
In general I agree, but not in this specific case. I think having `if` without curly braces and `else` with is kind of ugly.